README.org

```diff
@@ -78,7 +78,7 @@
 *** Main functions
```

Comment:
LOC 75 and 76, I'm not sure why it is spelled "aliase" with an "e" at the end. The correct spelling is: https://www.merriam-webster.com/dictionary/alias